### PR TITLE
Update function execute command

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -140,16 +145,20 @@ interval: The availability of the server is checked in the given interval
 
 #### example
 
-executeCommand("node", "node" ,{"args": ["-v"]})
+* executeCommand("node", "node" ,{"args": ["-v"]})
 Will create a command for executing node -v .
 
-executeCommand("somePollingScript.ps1","bash somePollingScript.sh", {"dir": "data/setup","asynchronous": true, "args": ["--params 5"]})
+* executeCommand("somePollingScript.ps1","bash somePollingScript.sh", {"dir": "data/setup","asynchronous": true, "args": ["--params 5"]})
```
[the remark this answers is not in the document]
Comment (dixyushi) [reply documentation/Functions.md]: done

File: documentation/Functions.md [hunk 2]
```diff
@@ -116,14 +116,19 @@ will set the workspace directory to "[working directory]/devonfw/workspaces/proj
 Learn more about the workspace directory and working directory on [Structure](https://github.com/devonfw-tutorials/tutorial-compiler/wiki/Structure)
 
 ### executeCommand <a name="executeCommand"></a>
+This function is used when you want to use a bash (or powershell/cmd on windows) command.
```
[the remark this answers is not in the document]
Comment (dixyushi) [reply documentation/Functions.md:119]: Edited it.

File: documentation/Functions.md [hunk 3]
```diff
@@ -140,16 +162,24 @@ interval: The availability of the server is checked in the given interval
...
+Will create a command to execute the script in the directory with the parameter --params 5 and in the current command prompt. The command prompt will be blocked until you stop the script.
 
-executeCommand("someServerScript.ps1","bash someServerScript.sh", {"asynchronous": true, "args":["-port 8080"] },{"port":8080 , "startupTime": 20, "path": "some/path/", "interval": 2})
+* executeCommand("someServerScript.ps1","bash someServerScript.sh", {"asynchronous": true, "args":["-port 8080"] },{"port":8080 , "startupTime": 20, "path": "some/path/", "interval": 2})
 Starting a server in a new terminal. You have to specify the port for testing, the other parameters are optional. The startupTime can specify how long the runner will wait for a response from the server process and with interval you can set the frequenzy for the server testing. The path is the subpath from your server that should be reached.
 
 
+Note: 
+
+1. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
```
Comment (GuentherJulian) [inline documentation/Functions.md:177]: Katacoda

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 3 above)
Comment (dixyushi) [reply documentation/Functions.md:177]: Changed!!

**Pull request merged**